test cases/common/58 run target/meson.build

```diff
@@ -31,7 +31,11 @@ run_target('upload2',
   depends : hex,
 )
 
-python3 = find_program('python3')
+python3 = find_program('python3', required : false)
+if not python3.found()
+  python3 = find_program('python')
+endif
+
 run_target('py3hi',
```

Comment:
Should definitely use the python3 module here since we're just running Python in a `run_target`